rust/agama-lib/share/profile.schema.json

```diff
@@ -303,41 +303,98 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
+        "boot": {
+          "$ref": "#/$defs/boot"
+        },
+        "drives": {
+          "title": "Drive devices",
+          "description": "Section describing drives (disks, BIOS RAIDs and multipath devices).",
+          "type": "array",
+          "items": {
+            "anyOf": [
+              {
+                "title": "Unpartitioned drive",
+                "description": "Drive without a partition table (e.g., directly formatted).",
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                  "search": {
+                    "description": "The search is limited to drives scope.",
+                    "$ref": "#/$defs/search"
+                  },
+                  "encryption": {
+                    "$ref": "#/$defs/encryption"
+                  },
+                  "filesystem": {
+                    "description": "The partition table (if any) is deleted.",
```

Comment:
Does this comment belong to `filesystem`?

Reply:
Maybe not. I will remove it.